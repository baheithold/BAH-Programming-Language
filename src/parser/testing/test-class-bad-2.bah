#This file tests an invalid class definition

class {
    var width;
    var height;
