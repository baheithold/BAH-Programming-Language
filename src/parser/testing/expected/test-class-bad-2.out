ERROR, line 3: expected ID_TYPE, but encountered OBRACE
